src/actor.jl

```diff
@@ -4,22 +4,53 @@ mutable struct Actor
     surface::Ptr{SDL2.Surface}
     position::Rect
     scale::Vector{Float64}
+    center::Vector{Int}
```

Comment:
Why do we need this attribute?

Reply:
Hi aviks, I've been using this attribute to rotate the actor about a center that is *not* its own center. eg a bird circling, things swirling down a drain, etc. 

Although, I can see how this could be a bit confusing... Maybe we could use a more descriptive name like `rotate_center`?

Reply:
I'd prefer to have this PR be only about text rendering, and discuss the rotation issue in a separate PR. How did you create this PR? Do you have a branch on your local disk with these changes?

Reply:
Sounds good to me--I probably shouldn't be mixing PRs like that.
 
I used to have a public fork of this project but seeing as my code has deviated significantly in the past few months and I'm targeting a marketable product, I've decided to close the branch for now. 

However, I do have other PRs planned for GameZero, including a function for capturing user text with the SDL2.StartTextInput() API. I currently have text inputs being captured into a composition string, but I need to implement a SDL2.Redraw() into my loop to update the screen and write functions for special events like 'enter', `esc`, directional arrow keys, etc. for editing / completing the composition. 

Happy to post a WIP PR for this if you like. :)